src/containers/Layout/Layout.js

```diff
@@ -23,6 +23,8 @@ const Layout = (props) => {
     setToggled(false);
   };
 
+  document.body.style.overflow = toggled ? "hidden" : "unset";
```

Comment:
hmmm... is the purpose of this to prevent scrolling?

Reply:
yep

Reply:
what's happening here now? lol what's changing

Reply:
oh hi guys, pwede ko pala kayo chat from here lol

Reply:
Lolz